NuGet pack and publish failing due to missing new React.Windows.Desktop folder in published artifacts from build

##[error]The nuget command failed with exit code(1) and error(Could not find a part of the path 'D:\a\1\s\ReactWindows\x86\Debug\React.Windows.Desktop'.

This is failing in main, 0.65-stable, and 0.64-stable and is therefore blocking all binary publishing.

--- a/.ado/publish.yml
+++ b/.ado/publish.yml
@@ -191,6 +191,7 @@ jobs:
           buildPlatform: $(BuildPlatform)
           buildConfiguration: $(BuildConfiguration)
           contents: |
+            React.Windows.Desktop\**
             React.Windows.Desktop.DLL\**
             React.Windows.Desktop.Test.DLL\**
       

